packages/components/src/custom-select-control-v2/custom-select.tsx

```diff
@@ -81,21 +75,18 @@ const UnconnectedCustomSelectButton = (
 			showOnKeyDown={ false }
 		>
 			<div>{ computedRenderSelectedValue( currentValue ) }</div>
-			<Icon icon={ chevronDown } size={ 18 } />
 		</Styled.Select>
 	);
 };
 
-const CustomSelectButton = contextConnectWithoutRef(
-	UnconnectedCustomSelectButton,
-	'CustomSelectControlButton'
-);
```

Comment:
We don't need to connect this to the context system anymore.

Reply:
Nevermind, already seen https://github.com/WordPress/gutenberg/pull/60261#discussion_r1542085948 👍🏻